forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt

```diff
@@ -5,9 +5,13 @@ PT:3/1
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConnive | TriggerDescription$ When CARDNAME enters the battlefield, it connives. (Draw a card, then discard a card. If you discarded a nonland card, put a +1/+1 counter on this creature.)
 SVar:TrigConnive:DB$ Connive
 T:Mode$ DiscardedAll | ValidPlayer$ You | ValidCard$ Card | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you discard one or more cards, exile them from your graveyard.
-SVar:TrigExile:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card.TriggeredCards
+SVar:TrigExile:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card.TriggeredCards | RememberChanged$ True
```

Comment:
Hmn ideally Remembered on ExiledWithSource would be redundant, I think it should have worked with it alone?

Maybe it's missing timestamp check?

Reply:
There are many cards that use this janky way of doing it.

Probably is related to timestamps... the issue iirc is that when you bounce something, its "exiled with" doesn't get cleared

Reply:
@Hanmac has mentioned a few times that he has a refactor of how Forge handles exiling in the works